### PR TITLE
Show the map but not the UI when hideUi=true

### PR DESCRIPTION
**Pull request by kring**
@meh9

### CODE REVIEW

File: lib/ReactViews/StandardUserInterface/map-column.scss [hunk 1]
```diff
@@ -18,7 +18,6 @@
 
 .map__cell {
   display: table-cell;
-  position: relative;
```
Comment (kring) [inline lib/ReactViews/StandardUserInterface/map-column.scss:21]: This was the key line that made the map reappear.  Hopefully removing it doesn't break IE or something crazy.  It seemed ok in my testing.  @AlexGilleran

**Comment by RacingTadpole** [comment]
Hi - happy to review this one but it looks like there's some background to it that I've missed - so I'm going to assign @AlexGilleran - hope that's ok!

**Comment by AlexGilleran** [comment]
Unfortunately this makes the timeline/graph pop up over the top of the map :(

image

image

**Comment by kring** [comment]
:scream: thought I tested that.  I'll look again.

**Comment by AlexGilleran** [comment]
It's a subtle one because somehow it moves the disclaimer but not the legend :/

**Comment by kring** [comment]
Ok this is ready now.

**Pull request merged**